versions/releases.py

```diff
@@ -163,7 +166,29 @@ def get_artifactory_download_data(url):
     }
 
 
-def get_release_notes_for_version(version_pk):
+def get_release_notes_for_version_s3(version_pk):
+    """Retrieve the adoc release notes from S3 and return the converted html string"""
+    try:
+        version = Version.objects.get(pk=version_pk)
+    except Version.DoesNotExist:
+        logger.info(
+            "get_release_notes_for_version_s3_error_version_not_found",
+            version_pk=version_pk,
+        )
+        raise Version.DoesNotExist
```

Comment:
A little cleaner, and keeps the original stack trace intact.

```suggestion
        raise
```